src/dotnet/APIView/APIViewWeb/Repositories/CosmosReviewRepository.cs

```diff
@@ -207,15 +207,13 @@ public async Task<IEnumerable<string>> GetReviewFirstLevelPropertiesAsync(string
                 }
                 else
                 {
-                    queryStringBuilder.Append($" AND (r.Author IN {searchAsQueryStr}");
+                    queryStringBuilder.Append($" AND (");
                     foreach (var item in search) 
                     {
                         var query = '"' + $"{item}" + '"';
-                        queryStringBuilder.Append($" OR CONTAINS(ARRAY_SLICE(r.Revisions, -1)[0].Name, {query}, true)");
```

Comment:
Does this mean that we will not find things that have the search query string in the revisions now, and is that a big deal?

Reply:
Name property in review level is package name. Name in the Revision has file name, and this is same in all cases. So, this won't impact search results when searching a review with a given package name. If anyone wants to find reviews submitted by a specific user, then filter `author:` can be used.